ton_client/client/src/contracts/deploy.rs

```diff
@@ -24,96 +25,116 @@ use ton_sdk::NodeClient;
 
 const DEFAULT_WORKCHAIN: i32 = 0;
 
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct DeployFunctionCallSet {
```

Comment:
I think it's very difficult name. Maybe rename to simple `DeployCall`.

Reply:
I don't think that it is a good idea because `DeployFunctionCallSet` is wrapper for `FunctionCallSet` struct on deploy call. I believe such name is quite suitable